edu_edfi_airflow/dags/earthbeam_dag.py

```diff
@@ -59,49 +56,10 @@ def __init__(self,
         self.pool = pool
         self.earthmover_pool = earthmover_pool or self.pool
         self.lightbeam_pool = lightbeam_pool or self.pool
-        self.slack_conn_id = slack_conn_id
 
         self.fast_cleanup = fast_cleanup
 
-        self.dag = self.initialize_dag(**kwargs)
-
-
-    def initialize_dag(self,
```

Comment:
Remove `initialize_dag()` helper, since this logic is encapsulated in `EACustomDAG`.